src/test/java/uk/gov/pay/api/service/CardPaymentSearchServiceTest.java

```diff
@@ -56,34 +56,7 @@ public void setUp() {
                 new PaginationDecorator(configuration),
                 new LedgerService(client, ledgerUriGenerator));
     }
-
-    @Test
-    @PactVerification({"ledger"})
-    @Pacts(pacts = {"publicapi-ledger-search-payment-by-cardholder-name"})
-    public void searchShouldReturnAResponseWithOneTransaction_whenFilteringByCardHolderNameFromLedger() {
```

Comment:
It's removed because I added cardholder search in https://github.com/alphagov/pay-publicapi/pull/855/commits/6469117f60c62aa56635c018e5e49ceff7547081#diff-12d44c70ea41c7574744a3ccc7ca7b55R17 and https://github.com/alphagov/pay-publicapi/blob/0be51fbc3484ac24e215f69ca16dac84fd90d3eb/src/test/java/uk/gov/pay/api/service/CardPaymentSearchServiceTest.java#L174 is already searching by cardholder name.